driverkit/utils/build

```diff
@@ -72,5 +72,8 @@ fi
 if [[ ${should_build} == "true" ]]; then
     mkdir -p output/${driver_version}/${arch}
     # if we have an architecture in our config path, force use it
+    if [[ -z ${DRIVERKIT_BACKEND} ]]; then
```

Comment:
Hi @Lowaiz, good catch! Thank you.

I'd rather use [bash expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) with default value, something like:
```bash
DRIVERKIT_BACKEND= ${DRIVERKIT_BACKEND:-docker}
```
from line 10